flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNSessionFIFOITCase.java

```diff
@@ -85,21 +91,56 @@ public void checkForProhibitedLogContents() {
 	public void testDetachedMode() throws InterruptedException, IOException {
 		LOG.info("Starting testDetachedMode()");
 		addTestAppender(FlinkYarnSessionCli.class, Level.INFO);
-		Runner runner =
-			startWithArgs(new String[]{"-j", flinkUberjar.getAbsolutePath(),
-						"-t", flinkLibFolder.getAbsolutePath(),
-						"-n", "1",
-						"-jm", "768",
-						"-tm", "1024",
-						"--name", "MyCustomName", // test setting a custom name
-						"--detached"},
+
+		File exampleJarLocation = getTestJarPath("StreamingWordCount.jar");
+		// get temporary file for reading input data for wordcount example
+		File tmpInFile = tmp.newFile();
+		FileUtils.writeStringToFile(tmpInFile, WordCountData.TEXT);
+
+		ArrayList<String> args = new ArrayList<>();
+		args.add("-j"); args.add(flinkUberjar.getAbsolutePath());
```

Comment:
😄  I see what you're trying to do here, but I'm fairly certain this will get auto-formatted at some point.

I would do it like this:
```
ArrayList<String> args = new ArrayList<>();
args.add("-j");
args.add(flinkUberjar.getAbsolutePath());

args.add("-t");
args.add(flinkLibFolder.getAbsolutePath());

...
```